src/Pages/HomePage/HomePage.js

```diff
@@ -0,0 +1,21 @@
+import "./HomePage.css"
+import { Sidebar } from "../../Components/Sidebar/Sidebar"
+import { Feedbar } from "../../Components/Feedbar/Feedbar"
+import { Aside } from "../../Components/Aside/Aside"
+import { BottomNav } from "../../Components/BottomNav/BottomNav"
+
```

Comment:
not mandatory but you can use [absolute path](https://dev.to/emanuelone/how-to-use-absolute-path-in-reactjs-503i#:~:text=Absolute%20Path%20is%20when%20you,located%20in%20the%20project%20structure.) here, it will look more clean